Cabal-tests/tests/ParserTests.hs

```diff
@@ -14,7 +14,19 @@ import Control.Monad                               (unless, void)
 import Data.Algorithm.Diff                         (PolyDiff (..), getGroupedDiff)
 import Data.Maybe                                  (isNothing)
 import Distribution.Fields                         (pwarning)
-import Distribution.PackageDescription             (GenericPackageDescription)
+import Distribution.PackageDescription
+  ( GenericPackageDescription
+      ( packageDescription
```

Comment:
In general it's more robust to `import Distribution.PackageDescription (GenericPackageDescription, fooField, bazField)` than `import Distribution.PackageDescription (GenericPackageDescription (fooField, bazField))`. If in future the fields of GPD are changed and old accessors become simple functions (retained for backwards compatibility), the first form survives without a change, while the second would have to be amended.